### PR TITLE
chore: Make dependency list compatible with npm (WEBAPP-5593)

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,43 +3,43 @@
   "license": "GPL-3.0",
   "dependencies": {
     "@babel/polyfill": "7.0.0",
-    "@bower_components/amplify": "wireapp/amplify#v1.1.7",
-    "@bower_components/antiscroll-2": "wireapp/antiscroll-2#1.3.1",
-    "@bower_components/bazinga64": "wireapp/bazinga64#5.0.3",
-    "@bower_components/bytebuffer": "dcodeIO/ByteBuffer.js#~3",
+    "@bower_components/amplify": "wireapp/amplify#v1.1.6",
```
Comment (ffflorian) [inline package.json]: Let's stay at v1.1.7.

File: package.json [hunk 2]
```diff
@@ -202,24 +202,24 @@
...
     "fix:assets": "yarn prettier --write && yarn stylelint",
     "fix:js": "yarn test:js --fix",
     "fix:ts": "yarn test:ts --fix",
-    "postinstall": "grunt init",
     "prettier": "prettier --ignore-path .gitignore \"**/*.{json,md}\"",
     "start": "concurrently \"yarn start:dev\" \"grunt\"",
     "start:dev": "webpack-dev-server --config webpack.config.dev.js --open",
     "start:server:dev": "cross-env \"NODE_ENV=development\" node ./aws/index.js",
-    "start:prod": "pm2 start \"./aws/index.js\" --name \"Webapp\" && pm2 logs",
+    "start:prod": "pm2 start \"./index.js\" --name \"Webapp\" && pm2 logs",
```
Comment (ffflorian) [inline package.json:223]: `outDir` is `./aws/`.

File: .ebextensions/node-settings.config [hunk 3]
```diff
@@ -0,0 +1,6 @@
+option_settings:
```
Comment (Yserz) [inline .ebextensions/node-settings.config]: that should be inside the `aws` folder

File: package.json (hunk 2 above)
Comment (bennycode) [reply package.json:223]: But aws gets extracted on ElasticBeanstalk, so there is no more `aws` directory on the server.

Comment (ffflorian) [reply package.json:223]: Ah, yes, of course.

**Pull request merged**